packages/atomic/src/components/atomic-search-interface/analytics-config.spec.ts

```diff
@@ -0,0 +1,108 @@
+import {
```

Comment:
there are quite a few warnings in this file, could they be avoided?

Reply:
Well add an eslint disable for `any`. Unfortunately, it's needed in this file. The types coming from `coveo.analytics` for the analytics middleware uses `any`.